chainer/links/caffe/caffe_function.py

```diff
@@ -547,18 +554,6 @@ def __call__(self, *xs):
         return self.func(xs, **self.kwargs)
 
 
-class _DropoutFunction(object):
```

Comment:
OK, no problem. We may need to mention that pickle files are not compatible in the document or the release note.